packages/yew/src/virtual_dom/vnode.rs

```diff
@@ -16,25 +17,27 @@ use crate::AttrValue;
 #[must_use = "html does not do anything unless returned to Yew for rendering."]
 pub enum VNode {
```

Comment:
May I ask why option 1 is selected by this pull request?

I feel other options would produce better public APIs.

Reply:
I think in the end we need both option 2 and 3. Because option 2 is better for the user as they will pass html nodes (Html/VNode) through props so this need to be fast. But option 3 is important for the internals of Yew. Huh but it's really just theory.

Reply:
Users may also want to pass VChild/VList directly for a better API to components